### PR TITLE
Implement `down` subcommand

### PR DESCRIPTION
**Pull request by nawok**
## Motivation

Partially brings back `railway down --yes` command (without the possibility to specify the environment—the linked one is used instead).

```
Remove the most recent deployment

Usage: railway down [OPTIONS]

Options:
  -y, --yes      Skip confirmation dialog
```

### CODE REVIEW

File: src/gql/mutations/mod.rs [hunk 1]
```diff
@@ -3,50 +3,50 @@ use graphql_client::GraphQLQuery;
...
 
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/gql/schema.graphql",
-    query_path = "src/gql/mutations/strings/PluginDelete.graphql",
+    query_path = "src/gql/mutations/strings/LoginSessionConsume.graphql",
     response_derives = "Debug, Serialize, Clone"
 )]
-pub struct PluginDelete;
+pub struct LoginSessionConsume;
```
Comment (nawok) [inline src/gql/mutations/mod.rs:17]: This is the actual addition. I reordered the mutations alphabetically in a [separate commit](https://github.com/railwayapp/cli/pull/339/commits/d860d697eb4281235f79c327672a3d4d49a202e6) so it is easier to decide where to put it. Sorry for a messy diff.

File: src/gql/mutations/mod.rs [hunk 2]
```diff
@@ -3,50 +3,50 @@ use graphql_client::GraphQLQuery;
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/gql/schema.graphql",
-    query_path = "src/gql/mutations/strings/PluginCreate.graphql",
+    query_path = "src/gql/mutations/strings/DeploymentRemove.graphql",
     response_derives = "Debug, Serialize, Clone"
 )]
-pub struct PluginCreate;
+pub struct DeploymentRemove;
```
Comment (nawok) [inline src/gql/mutations/mod.rs:9]: This is the actual addition. I reordered the mutations alphabetically in a [separate commit](https://github.com/railwayapp/cli/pull/339/commits/d860d697eb4281235f79c327672a3d4d49a202e6) so it is easier to decide where to put it. Sorry for a messy diff.

**Review by nawok — COMMENTED** [review]
Accidentally started a review of my own PR instead of adding the comment above ↑.

**Comment by Eejit43** [comment]
What is the status of this request? It would be amazing to get this merged as the lack of this command is the one thing holding me back from updating. :)

**Comment by JakeCooper** [comment]
I pulled it down but it doesn't seem like there's a `down` command?

CleanShot 2023-04-07 at 17 45 11@2x

File: src/gql/mutations/mod.rs (hunk 2 above)
Comment (JakeCooper) [reply src/gql/mutations/mod.rs:9]: All good thanks for doing that!

**Review by JakeCooper — APPROVED** [review]
Looks good gonna merge!

**Pull request merged**